spec/support/examples_group/shared_examples_for_provisioning_state_machine.rb

```diff
@@ -72,3 +72,9 @@
     expect(actual_states_with_counts).to eq(expected_states_with_counts)
   end
 end
+
+shared_examples_for "check class for platform" do |platform|
```

Comment:
@agrare @Fryguy Added the shared example, not sure if its the correct file to place it in but let me know, thanks.
https://github.com/ManageIQ/manageiq-providers-ovirt/blob/7aa1e58d6d71118ff808b7d24d2d1edc44676642/spec/models/manageiq/providers/ovirt/infra_manager/provision_workflow_spec.rb#L13